<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="User-PC" name="Student.AttemptSurvey" tests="9" failures="1" timestamp="8 May 2017 16:05:50 GMT" time="732.666" errors="0">
  <testcase name="SurveyData1" time="79.215" classname="Student.AttemptSurvey"/>
  <testcase name="SurveyData1" time="65.486" classname="Student.AttemptSurvey"/>
  <testcase name="SurveyData1" time="82.919" classname="Student.AttemptSurvey"/>
  <testcase name="SurveyData1" time="102.683" classname="Student.AttemptSurvey"/>
  <testcase name="SurveyData1" time="81.009" classname="Student.AttemptSurvey"/>
  <testcase name="SurveyData1" time="69.141" classname="Student.AttemptSurvey"/>
  <testcase name="SurveyData1" time="81.281" classname="Student.AttemptSurvey"/>
  <testcase name="SurveyData1" time="80.259" classname="Student.AttemptSurvey"/>
  <testcase name="SurveyData1" time="90.673" classname="Student.AttemptSurvey">
    <failure type="java.lang.AssertionError" message="Login Failed for:Sinpods0020 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Failed for:Sinpods0020 expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at Common.Login.LoginD(Login.java:35)
	at Common.Login.main(Login.java:89)
	at Student.AttemptSurvey.AttemptSurvey(AttemptSurvey.java:65)
	at Student.AttemptSurvey.SurveyData1(AttemptSurvey.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- SurveyData1 -->
</testsuite> <!-- Student.AttemptSurvey -->
